src/SignalR/common/SignalR.Common/src/IInvocationBinder.cs

```diff
@@ -32,4 +32,13 @@ public interface IInvocationBinder
     /// <param name="streamId">The ID of the stream the stream item is a part of.</param>
     /// <returns>The <see cref="Type"/> of the item the stream contains.</returns>
     Type GetStreamItemType(string streamId);
+
+#if NETCOREAPP
+    /// <summary>
+    /// Gets the <see cref="string"/> for the target byte.
+    /// </summary>
+    /// <param name="utf8Bytes">The target name as a utf8 sequence</param>
+    /// <returns>A string that represents the target.</returns>
```

Comment:
```suggestion
    /// <returns>A string that represents the target. Or null if the target string couldn't be determined. Fallback to normal string creation like Encoding.UTF8.GetString(bytes) in that case.</returns>
```

Reply:
There's no fallback internally. I would leave off that last part.